jdaviz/configs/default/plugins/viewers.py

```diff
@@ -36,25 +36,55 @@ def __init__(self, *args, **kwargs):
     @property
     def user_api(self):
         # default exposed user APIs.  Can override this method in any particular viewer.
+        expose = ['data_labels', 'data_labels_loaded', 'data_labels_visible']
```

Comment:
Hmm...

1. Does this include Mosviz table viewer? Do we want to expose these for Mosviz?
2. How do you want the Imviz fake WCS layers to behave for this? Are they not exposed here even though people can see them in the Data drop down?
3. I am guessing for Cubeviz case, the spectrum viewer will say it has no data labels even though a spectrum is visible because of the collapse magic?

Reply:
>Does this include Mosviz table viewer? Do we want to expose these for Mosviz?

Good point, I'll exclude from the user-API for table viewers

> How do you want the Imviz fake WCS layers to behave for this? Are they not exposed here even though people can see them in the Data drop down?

Good question, they will show up here , and maybe we will want to exclude them.  If that's what we want, I can do a follow-up PR to the image rotation PR once its rebased on this or merged into main.

> I am guessing for Cubeviz case, the spectrum viewer will say it has no data labels even though a spectrum is visible because of the collapse magic?

No, the spectrum should still show up as it is a data layer loaded into that viewer.

Reply:
Excluded now from mosviz, and confirmed that the cube shows up as expected in the spectrum viewer (and matching the UI as of #2631).

I also cross linked back here from the image rotation PR so that we can consider your second question then.